xml/System.Workflow.ComponentModel/Activity.xml

```diff
@@ -254,7 +254,7 @@ public class SendEmail : Activity
 ## Remarks  
  Raising an event invokes the event handler through a delegate. For more information, see [Handling and Raising Events](~/docs/standard/events/index.md).  
   
- The <xref:System.Workflow.ComponentModel.Activity.Cancel%2A> method also allows derived classes to handle the event without attaching a delegate. This is the preferred technique for handling the event in a derived class.  For more information about handling events, see [Consuming Events](https://msdn.microsoft.com/library/01e4f1bc-e55e-413f-98c7-6588493e5f67(v=vs.100)).  
+ The <xref:System.Workflow.ComponentModel.Activity.Cancel%2A> method also allows derived classes to handle the event without attaching a delegate. This is the preferred technique for handling the event in a derived class.  For more information about handling events, see [Handling and raising events](~/docs/standard/events/index.md).  
```

Comment:
```suggestion
 The <xref:System.Workflow.ComponentModel.Activity.Cancel%2A> method also allows derived classes to handle the event without attaching a delegate. This is the preferred technique for handling the event in a derived class. For more information about handling events, see [Handling and raising events](~/docs/standard/events/index.md).  
```

Reply:
Actually, given that previous paragraph already link to this topic, should we restructure these remarks to only link once?